dashboard/test/models/census/census_summary_test.rb

```diff
@@ -37,6 +37,90 @@ class Census::CensusSummaryTest < ActiveSupport::TestCase
     refute summary.valid?
   end
 
+  test "High school with NONE doesn't teach" do
+    submission = build :census_submission, how_many_20_hours: "NONE"
+    is_high_school = true
+    refute Census::CensusSummary.submission_teaches_cs?(submission, is_high_school)
+  end
```

Comment:
question: why calling out this `is_high_school` variable in each test? If it's for readability (my guess), then it might be better to convert that parameter to a keyword argument in the method definition:

```ruby
def self.submission_teaches_cs?(submission, is_high_school:)
  # ...
end
```
Which would be called like:
```ruby
Census::CensusSummary.submission_teaches_cs?(submission, is_high_school: true)
```

Reply:
Yes, it was just for readability. I'll change it based on your suggestion.